packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -1,31 +1,25 @@
 ---
-component: Reference
-title: API Reference
-top_links:
-  - name: API Concepts
-    path: /docs/reference/api-overview/
-    icon: icons/icon--docs-apiconcepts.svg
-  - name: Error Codes
-    path: /docs/reference/error-codes/
-    icon: icons/icon--docs-errorcodes.svg
-  - name: Okta Expression Language
-    path: /docs/reference/okta-expression-language/
-    icon: icons/icon--docs-expressionlang.svg
-  - name: Postman Collections
-    path: /docs/reference/postman-collections/
-    icon: icons/icon--docs-postman.svg
+title: Reference overview
 ---
 
-::: slot left
+# Reference overview
+
+Details on parameters, requests, and responses for Okta's API endpoints.
+
+
+Okta APIs generally fall two categories: signing in your users, and managing
+resources in your Okta org.
+
 ## Sign in Your Users
 API endpoints to authenticate your users, challenge for factors, recover passwords, and more. [Learn about which APIs to use.](/docs/concepts/oauth-openid/#authentication-api-vs-oauth-20-vs-openid-connect)
+API endpoints to authenticate your users, challenge for factors, recover passwords, and more.
+  - The [Authentication API](/docs/reference/api/authn) controls user accesss to Okta.
+  - The [OpenID Connect & OAuth 2.0 API](/docs/reference/api/oidc/) controls users access to your applications.
 
-<CategoryLinks category="authentication" class="list--with-descriptions" />
-:::
+[Learn about which APIs to use.](/docs/concepts/oauth-openid/#authentication-api-vs-oauth-20-vs-openid-connect)
 
-::: slot right
-## Manage Okta Resources
-REST endpoints to configure resources such as users, apps, sessions, and factors whenever you need.
+## Manage Okta Objects
+REST endpoints to configure resources such as users, apps, sessions, and factors
```

Comment:
REST endpoints ~to~ configure resources such as users, apps, sessions, and factors when needed.

Reply:
done